ReleaseTooling/Sources/ZipBuilder/main.swift

```diff
@@ -102,14 +102,15 @@ struct ZipBuilderTool: ParsableCommand {
 
   /// The minimum iOS Version to build for.
   @Option(default: "10.0",
-          help: ArgumentHelp("The minimum supported iOS version. The default is 10.0."))
+          help: ArgumentHelp("The minimum supported iOS version."))
   var minimumIOSVersion: String
 
   /// The list of architectures to build for.
   @Option(parsing: .upToNextOption,
           help: ArgumentHelp("""
-          The list of platforms to build for. The default list is \
-          \(TargetPlatform.allCases.map { $0.sdkName }).
+          The list of target platforms to build for. The default list is \
+          \(TargetPlatform.allCases.map { $0.sdkName }). Note that `macosx` is currently Catalyst \
```

Comment:
Output is 
`  --platforms <platforms> The list of target platforms to build for. The default list is ["iphoneos", "iphonesimulator", "macosx"]. Note that `macosx` is currently Catalyst only.`

The macosx was misleading before.